stories/va-accordion.stories.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
I'll see what I can do.

Reply:
I don't think that it is _impossible_, but the work to get this done seems like it is out of scope. Our stories on the `web-components` branch are written in typescript, so we can't just import them as they are into a branch with no typescript support. 

One option would be to run the `build-storybook` command as part of our build so that we get the built stories as JS. At this point we would end up with something like `storybook-static/main.abc123.bundle.js`, where all of our story exports are. Ignoring the problem of finding the right file to import based on the hashed filename (and there [isn't an easy way](https://storybook.js.org/docs/react/api/cli-options#build-storybook) to have the command build the stories in their own file), I expect that there would be additional problems here around conflicting story names. For example, each component will likely have a `Default` story, so how do we import N `Default`s from the same file?

With that being said, I think that a custom webpack configuration would allow us to get individual JS files per component that could be imported, but this is just a guess. I don't know how well a built story from one project would work when it is imported into another project and used as a story.

Another option would be if we reworked the `web-components` branch so that our stories were written as they are here (using React in plain `.jsx` files), and we could treat those source files in the `web-components` dependency as part of our Storybook instance. This would involve updating our configuration to [include stories from `node-modules/web-components`](https://github.com/department-of-veterans-affairs/component-library/blob/533d4b80aac29cce8eda2d27de36371ac4c181d2/.storybook/main.js#L2). Out of all the options this one seems like the least amount of effort, but I'm not sure right now if it would be a step forward.